### PR TITLE
1. Audio option removed for the template (Issue #2078), 2. Fb business manager verification link updated (Issue #2067)

### PR DESCRIPTION
**Pull request by enigmaticmahesh**
## Summary

**Issues Fixed:**

1. Audio option removed for the template (Issue [#2078](https://github.com/glific/glific-frontend/issues/2078))
2. Facebook Business Manager Verification Link Updated (Issue [#2067](https://github.com/glific/glific-frontend/issues/2067))
3. Link issue fixed in tooltip of chat contacts (Issue [#2074](https://github.com/glific/glific-frontend/issues/2074))
4. Character limit validation applied (Issue [#2066](https://github.com/glific/glific-frontend/issues/2066))

### CODE REVIEW

File: src/common/constants.ts [hunk 1]
```diff
@@ -116,7 +116,7 @@ export const is24HourWindowOver = (time: any) =>
 // connection retry attempt configuration
 export const CONNECTION_RECONNECT_ATTEMPTS = 5;
 
-export const MEDIA_MESSAGE_TYPES = ['IMAGE', 'AUDIO', 'VIDEO', 'DOCUMENT', 'STICKER'];
+export const MEDIA_MESSAGE_TYPES = ['IMAGE', 'VIDEO', 'DOCUMENT', 'STICKER'];
```
Comment (mdshamoon) [inline src/common/constants.ts:119]: I think this is used in multiple places like adding attachment and interactive messages. Please check if this is not creating an issue in other places.

**Review by mdshamoon — CHANGES_REQUESTED** [review]

File: src/common/constants.ts (hunk 1 above)
Comment (enigmaticmahesh) [reply src/common/constants.ts]: I fixed this in the [Commit](https://github.com/glific/glific-frontend/pull/2089/commits/f9d5009d5e181c43d6290f5ac83df3aef624dc8e)

**Review by mdshamoon — APPROVED** [review]
Looks good. Let's fix the test case in cypress also

**Comment by enigmaticmahesh** [comment]
> Looks good. Let's fix the test case in cypress also

Fixed in this [PR](https://github.com/glific/cypress-testing/pull/152)

**Pull request merged**